### DISCUSSION TITLE
Maybe we need a **Mark as Bad** label during validation

### DISCUSSION THREAD

**Discussion by taurenshaman** [idena-network/idena-docs#125] [IIPs]
In Epoch 0099, I met 5~6 similar flips:
A person, A white board, A final drawing.
Every thing are same, except the content on 4th flip: there are two single objects related to the flip tag.

Sorry, I forgot take screenshots...
I think the quality of these flips are very low, if there is a **Mark as Bad** label, I will give all of them.

**Comment by ubiubi18** [comment]
why not just report them?

**Reply by taurenshaman** [reply]
Because there are some more obviously invalid flips to report, and you may not have enough report places.
e.g. I met 5 to 6 similar flips in Epoch 0099, and there are 3 to 4 flips without logic, and I have only 6 or 7 report places.
